scripts/gcp_setup/environment/gcloud_env.py

```diff
@@ -741,40 +746,20 @@ def generate_deployment_templates(self):
 
         deploy_tpl_path = os.path.abspath(
             os.path.join(
-                self.DEPLOY_TPL_DIR_PATH,
+                self.ROOT_DIR_PATH,
+                'deployment-templates',
                 'deploy-forseti.yaml.in'))
         out_tpl_path = os.path.abspath(
             os.path.join(
-                self.DEPLOY_TPL_DIR_PATH,
+                self.ROOT_DIR_PATH,
+                'deployment-templates',
                 'deploy-forseti-{}.yaml'.format(os.getpid())))
-
-        # Ask for SendGrid API Key
-        sendgrid_api_key = raw_input(
-            'What is your SendGrid API key? (press [enter] to '
-            'leave blank) ').strip()
-        if sendgrid_api_key:
-            self.sendgrid_api_key = sendgrid_api_key
-
-        # Ask for notification sender email
-        notification_sender_email = raw_input(
-            'What is the sender email address for your notifications? '
-            '(press [enter] to leave blank) ').strip()
-        if notification_sender_email:
-            self.notification_sender_email = notification_sender_email
-
-        # Ask for notification recipient email
-        notification_recipient_email = raw_input(
-            'At what email address do you want to receive notifications? '
-            '(press [enter] to leave blank) ').strip()
-        if notification_recipient_email:
-            self.notification_recipient_email = notification_recipient_email
-
-        # Ask for GSuite super admin email
-        gsuite_super_admin_email = raw_input(
-            'What is your GSuite super admin\'s email? '
-            '(press [enter] to leave blank) ').strip()
-        if gsuite_super_admin_email:
-            self.gsuite_super_admin_email = gsuite_super_admin_email
+        rules_dir = os.path.abspath(
```

Comment:
No, I've removed that property. The user should not care how things are installed/stored on the VM.